### PR TITLE
*: pass column index to kv store.

### PR DESCRIPTION
**Pull request by hanfei1991**
In old implementation, we pass column id to kv storage. After open the switch of new planner, we can pass column index.
@zimulala PTAL

### CODE REVIEW

File: store/tikv/mock-tikv/executor.go [hunk 1]
```diff
@@ -607,20 +607,14 @@ func extractOffsetsInExpr(expr *tipb.Expr, columns []*tipb.ColumnInfo, collector
...
-				continue
-			}
-			if !isDuplicated(collector, idx) {
-				collector = append(collector, idx)
-			}
-			return collector, nil
+		if !isDuplicated(collector, int(idx)) {
+			collector = append(collector, int(idx))
 		}
-		return nil, errInvalid.Gen("column %d not found", i)
```
Comment (zimulala) [inline store/tikv/mock-tikv/executor.go]: It seems that `errInvalid` is useless, we can remove its definition.

File: store/tikv/mock-tikv/executor.go [hunk 2]
```diff
@@ -29,11 +29,6 @@ import (
...
 )
 
-// Error instances.
-var (
-	errInvalid = terror.ClassMockTikv.New(codeInvalid, "invalid operation")
-)
-
 // Error codes.
 const (
 	codeInvalid = 1
```
Comment (zimulala) [inline store/tikv/mock-tikv/executor.go]: Remove this useless code.

**Comment by zimulala** [comment]
LGTM

**Comment by hanfei1991** [comment]
@shenli PTAL

**Comment by shenli** [comment]
LGTM

**Pull request merged**